docs/proposal/rev_history.tex

```diff
@@ -8,7 +8,8 @@ \section*{Revision History}
 \hline
 Version \# & Initials & Release Date & Changes Made \\ \hline
 0.0 & MH & 2019-09-30 & Initial skeleton of the document.\\
-1.0 & MW & 2019-10-08 & First draft\\
+1.0 & MW & 2019-10-08 & First draft.\\
+1.0.1 & MH & 2019-10-09 & Revised ``Project Plan'' section.\\
```

Comment:
We should probably establish our versioning strategy ASAP. For drafts like this one I think we can stick at v1.0 (or better yet imo, 0.1).